entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if [[ ! "${SUPERUSER_PASSWORD}" == "" ]]; then
+    /opt/mumble/murmur.x86 -fg -ini ${CONFIG_PATH} -supw ${SUPERUSER_PASSWORD}
+    echo "Password for 'SuperUser' was set from environment variable"
+    unset SUPERUSER_PASSWORD
+fi
+
+exec $(echo "$@" | envsubst)
```

Comment:
```suggestion
exec /opt/mumble/murmur.x86 -fg -ini ${CONFIG_PATH}
```

Reply:
Like mentioned above, the intention here was also to allow a user running this container as an executable to pass any argument he likes. If that is not necessary we can change it to your suggestion and remove the `gettext` package as well.

Reply:
Running the container as an executable isn't even possible (mumble runs as a daemon), so that is not necessary.